src/command_modules/azure-cli-vm/azure/cli/command_modules/vm/_actions.py

```diff
@@ -72,8 +72,13 @@ def _load_images_from_publisher(publisher):
 
 
 def load_images_from_aliases_doc(publisher=None, offer=None, sku=None):
-    target_url = ('https://raw.githubusercontent.com/Azure/azure-rest-api-specs/'
-                  'master/arm-compute/quickstart-templates/aliases.json')
+    from azure.cli.core.cloud import get_active_cloud
+    CLOUD = get_active_cloud()
+    target_url = getattr(CLOUD.endpoints, 'vm_image_alias_doc', None)
```

Comment:
You should be able to do `CLOUD.endpoints. vm_image_alias_doc ` then catch the `CloudEndpointNotSetException` due to https://github.com/Azure/azure-cli/blob/master/src/azure-cli-core/azure/cli/core/cloud.py#L82.